distribution/src/main/assembly/generic-bin.xml

```diff
@@ -100,7 +100,7 @@
         <include>org.apache.xmlbeans:xmlbeans</include>
         <include>com.hazelcast:hazelcast</include>
         <include>com.hazelcast:hazelcast-client</include>
-        <include>com.hazelcast:hazelcast-cloud</include>
+        <include>com.hazelcast:hazelcast-aws</include>
```

Comment:
It is used for auto-discovery of cluster members on AWS. The gateway had a dependency for `hazelcast-cloud`, which was renamed to `hazelcast-aws`, so I assumed that this feature was needed. If you know that auto-discovery is not required, then I can remove it.